ttlcache.go

```diff
@@ -76,7 +76,8 @@ func (c *Cache) Count() int {
 
 // Close frees up resources used by the cache.
 func (c *Cache) Close() {
-	if c.IsOpen() {
+	wasOpen := c.open.Swap(false)
+	if wasOpen.(bool) {
```

Comment:
The constructor initializes the value for c.open; this could stand a cast error check, as we expose Cache{}